src/pages/Portfolio/ImpactDashboard/LoanCommentModal.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+	<kv-lightbox
+		:visible="loan.visible"
+		:title="modalTitle"
+		@lightbox-closed="closeModal"
+	>
+		<div class="tw-text-center">
```

Comment:
It looks like this is some duplicate code from the `CommentAsk` component. Would it make sense to abstract this out and use in both cases? I'm not sure how visually and/or functionally different these two components are.

Reply:
With the fix in the comment above, for the error message this is a little bit different, but we can abstract the submit method into a mixin to use it in both cases.

Reply:
Probably isn't needed in this changeset, but it would probably be worth creating a new ticket to cover abstracting out this deplicate functionality.